.github/actions/set_yarn_berry/action.yml

```
@@ -0,0 +1,13 @@
+name: Set Yarn Berry
+description: Enable Yarn Berry in the workflow
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Enable corepack
+      run: corepack enable
```

Nit: corepack will no longer be distributed by Node.js soon(after 25). It's better to install this
```suggestion
      run: npm install -g corepack
```

---

Cool, I would not worry about this now, since we pin Node 20, and Node 18 is still in the progress of deprecation...